projects/packages/autoloader/tests/php/tests/unit/ManifestGeneratorTest.php

```diff
@@ -26,18 +26,18 @@ public function test_manifests_do_nothing_without_content() {
 	 * Tests that manifests for classmaps are generated correctly.
 	 */
 	public function test_builds_classmap_manifest() {
-		$expected = <<<EXPECTED_FILE
+		$expected = <<<'EXPECTED_FILE'
 <?php
 
 // This file `test-file.php` was auto generated by automattic/jetpack-autoloader.
 
-\$vendorDir = dirname(__DIR__);
-\$baseDir   = dirname(\$vendorDir);
+$vendorDir = dirname(__DIR__);
```

Comment:
I'll just note it here, but applicable elsewhere too...I assume we don't care that this is not resulting in PHPCS-compliant code?